docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -94,9 +94,108 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Installation of generated systemd unit files.
+
+Podman generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of the systemctl(1) during installation.
+
+Once you have generated the systemd unit file. Copy the generated systemd file to ```/usr/lib/systemd/system``` for installing as a root user and to ```/usr/lib/systemd/user``` for installing it as a non-root user.
```

Comment:
```suggestion
Once you have generated the systemd unit file, you can copy the generated systemd file to ```/usr/lib/systemd/system``` for installing as a root user and to ```/usr/lib/systemd/user``` for installing it as a non-root user.
```

Reply:
Copying to `/usr/lib/systemd/user` requires root privileges which defeats the rootless purpose. Instead, we should recommend `$HOME/.config/systemd/user/`.

Reply:
Agreed. Fixed it.